pkg/ddc/jindo/transform.go

```diff
@@ -663,6 +664,18 @@ func (e *JindoEngine) transformLabels(runtime *datav1alpha1.JindoRuntime, value
 	return nil
 }
 
+func (e *JindoEngine) transformHostNetWork(runtime *datav1alpha1.JindoRuntime, value *Jindo) (err error) {
+	// to set hostnetwork
+	if runtime.Spec.DisableHostNetWork {
+		value.UseHostNetwork = false
+		value.UseHostPID = false
```

Comment:
Why setting PID along with network mode?

Reply:
ok，and useHostPID will default to true and current not to set by user